imap_processing/ialirt/packet_definitions/ialirt.xml

```diff
@@ -0,0 +1,805 @@
+				<xtce:LongDescription>Electron counts for CEM4 energy 2.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM4_E3" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM4 energy 3.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM4_E4" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM4 energy 4.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM5_E1" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM5 energy 1.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM5_E2" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM5 energy 2.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM5_E3" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM5 energy 3.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM5_E4" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM5 energy 4.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM6_E1" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM6 energy 1.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM6_E2" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM6 energy 2.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM6_E3" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM6 energy 3.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM6_E4" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM6 energy 4.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM7_E1" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM7 energy 1.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM7_E2" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM7 energy 2.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM7_E3" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM7 energy 3.</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SWE_CEM7_E4" parameterTypeRef="uint8">
+				<xtce:LongDescription>Electron counts for CEM7 energy 4.</xtce:LongDescription>
+			</xtce:Parameter>
+			<!-- SWE -->
+			<!-- MAG -->
+			<xtce:Parameter name="MAG_ACQ" parameterTypeRef="uint48">
+				<xtce:LongDescription>MAG Acquisition Time</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="MAG_STATUS" parameterTypeRef="uint24">
+				<xtce:LongDescription>MAG Status</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="MAG_DATA" parameterTypeRef="uint24">
+				<xtce:LongDescription>MAG Data</xtce:LongDescription>
+			</xtce:Parameter>
```

Comment:
There are more bits in the MAG Acquisition Time than the data?

I think the data fields might need to be BinaryParameterType / BYTE so you can unpack the bits once you get more info later.

Reply:
I'm going off of the spreadsheet being tracked on IMAPCATS-4:
![Screen Shot 2023-11-07 at 3 50 27 PM](https://github.com/IMAP-Science-Operations-Center/imap_processing/assets/46567335/f7d814f3-d281-4e4a-ae45-7058eb31f466)

Reply:
Interesting! That just seems odd to me, but you're definitely right. One thing I did just notice though is that it looks like the I-ALiRT spreadsheets have MET coming down, which is 32 bits instead of ACQ_TM which is 48 bits. Is there another mismatch there that we need to update in the I-ALiRT spreadsheet as well?

@maxinelasp did you have any conversations with MAG about their I-ALiRT data formatting?

Reply:
Yes. I had to piece together different spreadsheets to get the xml:
1. I-ALiRT Packet Definitions 2022_10_25.xlsx (old)
2. IALiRT SC Packet Content.xlsx (recent)
3. And then what was in IMAPCATS

So you are correct. The definitions from I-ALiRT Packet Definitions 2022_10_25.xlsx need to be updated.